theme/static/js/hs-vue/subjet-keywords.js

```diff
@@ -58,12 +58,12 @@ let subjKeywordsCmp = new Vue({
               {value: newVal.join(",")}, function (resp) {
                   if (resp.status === "success") {
                       vue.resKeywords = newVal;
+                      if (!newVal.length) {
+                          // If no keywords, the metadata is no longer sufficient to make the resource public
```

Comment:
what is the best practice in JS/jQuery/Vue for testing for undefined, null, empty string etc... guessing what you're doing here is fine

Reply:
That is always a bit difficult to memorize, but I do keep in mind some conditional rules that I often use. Like empty string, zero, etc being equivalent to `false` when used as conditionals.